.circleci/config/jobs/operator_upgrade_tests.yml

```diff
@@ -175,21 +175,18 @@ steps:
         SLEEP_SECONDS_BETWEEN_ATTEMPTS=5
         # total = 10 minutes wait time
 
-        EXPECTED_TAG="${LATEST_TAG}-ubi8"
         # Periodically poll if the snyk-monitor has upgraded
         for (( attempt=1; attempt<ATTEMPTS; attempt++))
         do
-          # Grab the tag of the snyk-monitor container image.
-          SNYK_MONITOR_POD=$(kubectl get pods -n snyk-monitor --no-headers | grep "snyk-monitor")
-          if [[ ! -z "${SNYK_MONITOR_POD}" ]]; then
```

Comment:
So we don't need to check `SNYK_MONITOR_POD` might be empty then?

Reply:
I just reverted to the old way as it used to work 🤷